apis/backupdr/v1alpha1/managementserver_types.go

```diff
@@ -0,0 +1,199 @@
+	ExternalRef *string `json:"externalRef,omitempty"`
+
+	// ObservedState is the state of the resource as most recently observed in GCP.
+	ObservedState *BackupDRManagementServerObservedState `json:"observedState,omitempty"`
+}
+
+// BackupDRManagementServerObservedState is the state of the BackupDRManagementServer resource as most recently observed in GCP.
+// +kcc:proto=google.cloud.backupdr.v1.ManagementServer
+type BackupDRManagementServerObservedState struct {
+	// Output only. Identifier. The resource name.
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.name
+	// NOTYET: this field serves the same purpose as externalRef
+	// Name *string `json:"name,omitempty"`
+
+	// Output only. The time when the instance was created.
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.create_time
+	CreateTime *string `json:"createTime,omitempty"`
+
+	// Output only. The time when the instance was updated.
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.update_time
+	UpdateTime *string `json:"updateTime,omitempty"`
+
+	// Output only. The hostname or ip address of the exposed AGM endpoints, used
+	//  by clients to connect to AGM/RD graphical user interface and APIs.
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.management_uri
+	ManagementURI *ManagementURIObservedState `json:"managementURI,omitempty"`
+
+	// Output only. The hostnames of the exposed AGM endpoints for both types of
+	//  user i.e. 1p and 3p, used to connect AGM/RM UI.
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.workforce_identity_based_management_uri
+	WorkforceIdentityBasedManagementURI *WorkforceIdentityBasedManagementURIObservedState `json:"workforceIdentityBasedManagementURI,omitempty"`
+
+	// Output only. The ManagementServer state.
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.state
+	State *string `json:"state,omitempty"`
+
+	// Output only. The OAuth 2.0 client id is required to make API calls to the
+	//  BackupDR instance API of this ManagementServer. This is the value that
+	//  should be provided in the 'aud' field of the OIDC ID Token (see openid
+	//  specification
+	//  https://openid.net/specs/openid-connect-core-1_0.html#IDToken).
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.oauth2_client_id
+	OAUTH2ClientID *string `json:"oauth2ClientID,omitempty"`
+
+	// Output only. The OAuth client IDs for both types of user i.e. 1p and 3p.
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.workforce_identity_based_oauth2_client_id
+	WorkforceIdentityBasedOAUTH2ClientID *WorkforceIdentityBasedOAuth2ClientIDObservedState `json:"workforceIdentityBasedOAUTH2ClientID,omitempty"`
+
+	// Output only. The hostname or ip address of the exposed AGM endpoints, used
+	//  by BAs to connect to BA proxy.
+	// +kcc:proto:field=google.cloud.backupdr.v1.ManagementServer.ba_proxy_uri
+	BaProxyURI []string `json:"baProxyURI,omitempty"`
```

Comment:
Do you know what is "BA"? Also, shall this be plural ?

Reply:
Renamed to `BAProxyURIs` (plural).